source/_docs/user-dashboard.md

```diff
@@ -31,6 +31,10 @@ This is where you can update and manage your personal account details. Use the o
 - Add [machine tokens](/docs/machine-tokens/)  
 - [Delete your account](/docs/delete-account/)
 
+### Gravatar
+To associate an image with your Pantheon account, you'll need to [create a Gravatar](http://en.gravatar.com/) for the email address you use with your Pantheon account. If you already have a Gravatar set up for another email address, update your Gravatar profile to add the email address you use on Pantheon. For help, visit the [Gravatar Support](https://en.gravatar.com/support/) page.
```

Comment:
Maybe we should like to https://gravatar.com and let Gravatar redirect? I'm thinking en.gravatar.com is their english site and they have other sites that they'll redirect to based on the visitor's location?